qhub_ops/template/{{ cookiecutter.repo_directory }}/image/Dockerfile.jupyterlab

```diff
@@ -34,5 +44,21 @@ RUN groupadd --gid ${NB_UID} ${NB_USER} && \
     --uid ${NB_UID} \
     ${NB_USER}
 
+RUN chown -R ${NB_UID}:${NB_UID} /opt/conda/
```

Comment:
When qhub runs the jupyterlab user they will not have this uid/gid. It could be anything so I don't see a need to change the permissions of the /opt/conda directory.

Reply:
If this step is missing conda is not initialize for bash shell

Reply:
The user here is `jovyan` and group is `1000` .
This PR follows https://github.com/Quansight/jupyterhub-datum/blob/staging/deployments/datum/image/Dockerfile
and https://hub.docker.com/r/continuumio/miniconda3/dockerfile